src/pybind/mgr/cephadm/migrations.py

```diff
@@ -46,8 +45,28 @@ def set(self, val: int) -> None:
         self.mgr.set_module_option('migration_current', val)
         self.mgr.migration_current = val
 
+    def set_sane_migration_current(self) -> None:
+        # migration current should always be an integer
+        # between 0 and LAST_MIGRATION (inclusive) in order to
+        # actually carry out migration. If we find
+        # it is None or too high of a value here we should
+        # set it to some sane value
+        if self.mgr.migration_current is None:
+            logger.info('Found migration_current of "None". Setting to last migration.')
+            self.set(LAST_MIGRATION)
+
+        mc: Optional[int] = self.mgr.migration_current
+        if mc is not None and mc > LAST_MIGRATION:
```

Comment:
Unless I'm missing something you shouldn't need to check for None here, see my comment above